### PR TITLE
Footer added #63

### PR DESCRIPTION
**Pull request by Annu117**
**What kind of change does this PR introduce?**

The footer section was missing.

**Issue Number:**

#63

**Snapshots/Videos:**

image

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.footer {
+    position: relative; /* Ensure positioning for pseudo-element */
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    background-color: #f8f8f8;
```
Comment (Doraemon012) [inline src/components/Footer/Footer.css:6]: Please use the colors used in `theme.tsx`.
For example, 
``` css
 background-color: theme.colors.brightBackground;
```

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,76 @@
+.footer {
+    position: relative; /* Ensure positioning for pseudo-element */
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    background-color: #f8f8f8;
+    border-top: 1px solid #e7e7e7;
+}
+
+/* Add light blue strip at the top of footer */
```
Comment (Doraemon012) [inline src/components/Footer/Footer.css:10]: Remove the comments.

File: src/components/Footer/Footer.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    height: 4px; /* Height of the strip */
+    background-color: #5AB2FF; /* Light blue color */
+}
+
+/* Adjusted footer layout */
+.footer-about {
+    flex: 1;
+    margin-right: 20px;
+    margin-left: 50px;
+    font-size: 14px;
```
Comment (Doraemon012) [inline src/components/Footer/Footer.css:26]: Do not use font size in `px`, instead user `rem`. This helps to keep the text in the web[page responsive to different screen sizes. Use the font sizes defined in `theme.tsx`.

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes

**Comment by Doraemon012** [comment]
@Annu117 Please use [MUI](https://mui.com/material-ui/)

**Comment by Doraemon012** [comment]
@Annu117 `theme.tsx` has some change so, first, update your branch like this:
To resolve this, go to your branch, then click on `Sync Fork` and then click on `Update branch`.

You can use the below screen shot and recording as reference. Here, the branch `patch-1` is one commit ahead and 9 commits behind of the remote repository.

image

https://github.com/Tanay-ErrorCode/lupo-skill/assets/116624667/6fe638ca-bc95-4a5e-a43a-11d7757f21dd

**Comment by Annu117** [comment]
By mistake closed the pull will reopen after changes made.

File: src/components/Footer/Footer.css (hunk 1 above)
Comment (Annu117) [reply src/components/Footer/Footer.css]: Do u want me to create .scss file? .css file can't load .tsx file components.

File: src/components/Footer/Footer.tsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+);
+
+const Footer: React.FC = () => {
+  return (
+    <Box component="footer" className="footer">
+      <Box className="strip" />
+      <Container maxWidth="lg">
+        <Grid container spacing={4}>
+          <Grid item xs={12} md={6}>
+            <Typography variant="h6" gutterBottom className="footer-text">
```
Comment (Doraemon012) [inline src/components/Footer/Footer.tsx:37]: Make the headings bold.

File: src/components/Footer/Footer.tsx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+              Lupo Skill is a platform where you can share your skills by hosting events and learn new ones by joining others. Enjoy real-time updates and a fully responsive design, all with an easy-to-use interface.
+            </Typography>
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <Typography variant="h6" gutterBottom className="footer-text">
+              Our Team
+            </Typography>
+            <Box className="footer-text">
+              <LinkedInIcon />
+                Tanay
```
Comment (Doraemon012) [inline src/components/Footer/Footer.tsx:50]: Add the links to the text too.

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Doraemon012** [comment]
@Annu117 Good work !
Add some margin from the footer top, keep it around`6rem`. Also, increase the vertical padding inside the footer.

File: src/components/Footer/Footer.tsx (hunk 5 above)
Comment (Annu117) [reply src/components/Footer/Footer.tsx]: Done

File: src/components/Footer/Footer.tsx (hunk 4 above)
Comment (Annu117) [reply src/components/Footer/Footer.tsx]: Done

**Comment by Annu117** [comment]
changes save. Anything else left @Doraemon012 ?

**Comment by Annu117** [comment]
@Doraemon012 is there any issue? Or do you need a dark theme?

File: src/components/Footer/Footer.tsx [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+          </Grid>
+          <Grid item xs={12} md={3}>
+            <Typography variant="h6" gutterBottom className="footer-text" style={{ fontWeight: 'bold' }}>
+              Our Team
+            </Typography>
+            <Box className="footer-text">
+              <LinkedInLink url="https://www.linkedin.com/in/errorcode/" name="Tanay" />
+            </Box>
+            <Box className="footer-text">
+              <LinkedInLink url="https://www.linkedin.com/in/neyati/" name="Neyati" />
```
Comment (Doraemon012) [inline src/components/Footer/Footer.tsx:56]: Its `https://www.linkedin.com/in/neyati-iiit/`

File: src/components/Footer/Footer.tsx [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    {name}
+  </Link>
+);
+
+const Footer: React.FC = () => {
+  return (
+    <Box component="footer" className="footer">
+      <Box className="strip" />
+      <Container maxWidth="lg">
+        <Grid container spacing={4}>
```
Comment (Doraemon012) [inline src/components/Footer/Footer.tsx:39]: Make it `spacing={24}`

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by Annu117** [comment]
@Doraemon012 done

**Comment by Annu117** [comment]
@Doraemon012, plz merge.

**Review by Doraemon012 — APPROVED** [review]

**Comment by Tanay-ErrorCode** [comment]
@Annu117 Good work!

**Comment by Annu117** [comment]
Thank you:)

**Pull request merged**